src/Interfaces.ts

```diff
@@ -32,7 +32,7 @@ export type IResolverObject = {
   [key: string]: GraphQLFieldResolver<any, any> | IResolverOptions;
 };
 export interface IResolvers {
-  [key: string]: (() => any) | IResolverObject | GraphQLScalarType;
+  [key: string]: (() => any) | IResolverObject | GraphQLScalarType | any;
```

Comment:
Perhaps this any is a bit too open? (Looks like it triggers a bunch of additional errors later)

Reply:
Could this just be `GraphQLEnumType`?

Reply:
I don't think so, that's a full class and really this wants to represent the custom value lookup system which is currently   `[key: string]: string`.

I think I'm gonna go with:

```js
export type IEnumResolver = { [key: string]: string };
export interface IResolvers {
  [key: string]: (() => any) | IResolverObject | GraphQLScalarType | IEnumResolver;
}
```

Which tightens it, and doesn't create the extra errors in tests